Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -336,14 +336,86 @@ public protocol CarPlayManagerDelegate: AnyObject, UnimplementedLogging, CarPlay
      instance of either `CarPlayMapViewController` or `CarPlayNavigationViewController`.
      - returns: A `LineLayer` that is applied as a casing around the route line.
      
-     - seealso: `NavigationMapViewDelegate.navigationMapView(_:routeCasingLineLayerWithIdentifier:sourceIdentifier:)`,
-     `NavigationViewControllerDelegate.navigationViewController.carPlayManager(_:routeCasingLineLayerWithIdentifier:sourceIdentifier:)`.
+     - seealso: `CarPlayNavigationViewControllerDelegate.carPlayNavigationViewController(_:routeCasingLineLayerWithIdentifier:sourceIdentifier:)`,
+     `CarPlayMapViewControllerDelegate.carPlayMapViewController(_:routeCasingLineLayerWithIdentifier:sourceIdentifier:)`.
      */
     func carPlayManager(_ carPlayManager: CarPlayManager,
                         routeCasingLineLayerWithIdentifier identifier: String,
                         sourceIdentifier: String,
                         for parentViewController: UIViewController) -> LineLayer?
     
+    /**
+     Asks the receiver to return a `LineLayer` for highlighting restricted areas portions of the route,
+     given a layer identifier and a source identifier.
+     This method is invoked when the map view loads and any time routes are added.
+     
+     - parameter carPlayManager: The `CarPlayManager` object.
+     - parameter identifier: The `LineLayer` identifier.
+     - parameter sourceIdentifier: Identifier of the source, which contains the route data that this method would style.
+     - parameter parentViewController: The view controller that contains the map view, which is an
+     instance of either `CarPlayMapViewController` or `CarPlayNavigationViewController`.
+     - returns: A `LineLayer` that is applied as restricted areas on the route line.
+     
+     - seealso: `CarPlayNavigationViewControllerDelegate.carPlayNavigationViewController(_:routeRestrictedAreasLineLayerWithIdentifier:sourceIdentifier:)`,
+     `CarPlayMapViewControllerDelegate.carPlayMapViewController(_:routeRestrictedAreasLineLayerWithIdentifier:sourceIdentifier:)`.
+     */
+    func carPlayManager(_ carPlayManager: CarPlayManager,
```

Comment:
Also add the missing delegate for restricted areas in carPlay, along with the existing methods for route line and route casing.